src/lib/core/components/filter/HistogramFilter.tsx

```diff
@@ -791,6 +791,13 @@ function computeBinSlider(
       return { min: 1, max: 60, step: 1 };
     }
     case 'integer':
```

Comment:
Two things: For consistent code styling, maybe put curly braces around the integer case code (to match the others)?  And also, I'm curious why we always start with the min size rather than some aesthetically pleasing number of bins e.g. 10? (assuming it doesn't violate the min/max).

Reply:
Thanks Ryan - completely missed the curly braces - surprised it worked without!

Actually we don't always start with min bin size.  The recommended binWidth is supplied with variable metadata and we use that.  These calculations just provide the limits and step-size for the slider.